### ISSUE TITLE
RPC_PORT not being used

### ISSUE THREAD

**Issue by warrick-eosny** [EOSIO/ual-plainjs-renderer#28]
I've got a .env that looks as follows and it seems like the RPC_PORT is not being used when making the get_account request. 

```
CHAIN_ID=cf057bbfb72640471fd910bcb67639c22df9f92470936cddc1ade0e2f2e7dc4f
RPC_PROTOCOL=https
RPC_HOST=api.pennstation.eosnewyork.io
RPC_PORT=7101
``` 
The URL being called is: https://api.pennstation.eosnewyork.io/v1/chain/get_account (note the missing port 7101)

Also I'm not sure if this is a problem with core or just the example, but when it fails like this it seems to put the client into an endless loop of polling the server. With many users connected to a dapp it seems like could cause problems when a failure occurs.

![Snag_3c9b70e1](https://user-images.githubusercontent.com/40346010/55502152-ec4ab180-5619-11e9-8011-0f63f114a5fe.png)

**Comment by pingustar** [comment]
you can fix that by adding the port here:

https://github.com/EOSIO/ual-plainjs-renderer/blob/master/examples/src/ButtonWebView.ts#L95

like so:

```
const rpc = new JsonRpc(`${EXAMPLE_ENV.RPC_PROTOCOL}://${EXAMPLE_ENV.RPC_HOST}:${EXAMPLE_ENV.RPC_PORT}`)
```

**Comment by TaraTritt** [comment]
Closed by #37
